cmd/influxd/server_integration_test.go

```diff
@@ -157,6 +158,18 @@ func createCombinedNodeCluster(t *testing.T, testName, tmpDir string, nNodes int
 
 	}
 
+	// Sanity check that we created a cluster and data nodes have unique ids
+	ids := []int{}
```

Comment:
Would it be clearer if you just did something like this:
```
ids := make(map[int]bool)
for _, n := range nodes {
    ids[int(n.node.DataNode.ID())] = true
}
if len(ids) != len(nodes) {
    t.Fatalf("failed to create valid cluster")
}
```
No sorting required.